### ISSUE TITLE
Create github pull request

### ISSUE THREAD

**Issue by Odyno** [fabioz/mu-repo#22]
It would be very useful to have a way to create github/bitbucket pull requests on all the projects automatically

**Comment by Swivelgames** [comment]
Not to add too much to the list, but a gitlab merge request would be great too. I'm not sure about github.com or bitbucket, but the push command on gitlab actually provides a URL in the output. Even so, I think the API is relatively straightforward for these. It'd be an awesome feature, especially the ability to specify which destination branch to use for those of us that use release/iteration/multiple+integration branches in our projects.

**Comment by fabioz** [comment]
This was made available in 1.5.0 through the `mu open-url` command.

See: http://fabioz.github.io/mu-repo/open_url/

Note that there's a fix for 1.6.0 (because it didn't support relative repos properly, although if using from a parent directory where children are the repos it already did work in 1.5.0).

**Comment by fabioz** [comment]
p.s.: I haven't checked the gitlab url for the pull request, but if someone points me to it I'll update http://fabioz.github.io/mu-repo/open_url/

**Comment by Swivelgames** [comment]
@fabioz Here's an example URL:

```
https://gitlab.com/foobar/baz-repo/merge_requests/new?merge_request[source_branch]=my-feature-branch&merge_request[target_branch]=master
```

Yes, the brackets (`[` and `]`) are literal characters!

**Comment by Swivelgames** [comment]
So, I'm assuming it would look something like this:

```
https://gitlab.com/foobar/{repo}/merge_requests/new?merge_request[source_branch]={source}&merge_request[target_branch]={dest}
```

😃

**Comment by fabioz** [comment]
Thanks, just updated http://fabioz.github.io/mu-repo/open_url/

Also, just released 1.6.0 (so it can be pip installed).
